client/ayon_core/pipeline/template_data.py

```diff
@@ -101,6 +101,10 @@ def get_folder_template_data(folder_entity, project_name):
     return {
         "folder": {
             "name": folder_name,
+            "path": path,
+            "type": folder_entity["folderType"],
+            "parents": hierarchy,
+            "parent": parent_name,
```

Comment:
Is there really use in having the `parent` data duplicated outside the folder key and inside?

Reply:
I would say tha the data which are now in root of dictionary should be considered as deprecated with these changes.

Reply:
Just to check I get it correctly. 
these data should be considered as deprecated
![image](https://github.com/ynput/ayon-core/assets/20871534/9859da29-a8f1-471e-aff7-ecc26d9bd3c1)